.circleci/config.yml

```diff
@@ -422,13 +418,7 @@ jobs:
     steps:
       - run:
           name: Trigger UI GitHub Actions workflow
-          command: |
-            curl -X POST \
-              -H "Authorization: token $GITHUB_TOKEN" \
-              -H "Accept: application/vnd.github.v3+json" \
-              https://api.github.com/repos/ministryofjustice/hmpps-temporary-accommodation-ui/actions/workflows/generate-types.yml/dispatches \
-              -d '{"ref":"main"}'        
-
+          command: "curl -X POST \\\n  -H \"Authorization: token $GITHUB_TOKEN\" \\\n  -H \"Accept: application/vnd.github.v3+json\" \\\n  https://api.github.com/repos/ministryofjustice/hmpps-temporary-accommodation-ui/actions/workflows/generate-types.yml/dispatches \\\n  -d '{\"ref\":\"main\"}'        \n"
```

Comment:
This change is a bit concerning -- lots of escaped new lines?

Reply:
@Rosie-Brigham, @daveawc and i did think about this while working on the refactoring how swagger docs are generated.  I agree, we should switch it but we should wait until we can deploy the API in GH first as the new swagger docs will need the the api to be deployed to dev before generate-types can run

I'm going to rollback these changes specific lines of changes that the migration scripe made. I think it was `yq` was just trying to be smart